.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,51 @@
+name: Lint and Test Charts
+
+on:
+  push:
+    branches-ignore:
+    - 'main'
+    - 'repository'
+    
+  pull_request:
+    branches:
+    - '**'
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.1
```

Comment:
This is pretty old, could we bump it to the latest version?

Reply:
Sure.